Marlin/src/module/motion.cpp

```diff
@@ -1599,6 +1602,13 @@ void homeaxis(const AxisEnum axis) {
     if (axis == Z_AXIS && STOW_PROBE()) return;
   #endif
 
+  #ifdef HOMING_BACKOFF_MM
+    constexpr float endstop_backoff[XYZ] = HOMING_BACKOFF_MM;
+    float backoff_pos[XYZ]; COPY(backoff_pos, current_position);
+    backoff_pos[axis] -= ABS(endstop_backoff[axis]) * axis_home_dir;
+    do_blocking_move_to(backoff_pos);
+  #endif
```

Comment:
This probably won't be good on Delta. On delta machines we do not want to back off Z (all three towers) until all three axes have gone through their individual homing and bumping procedures. That is one of the considerations in the existing implementation. Blocking moves operate in cartesian space, so this will definitely do bad things on delta.

Reply:
Making the case more on more on why I should go buy a Delta.... Anyone care to donate one? Lol

Reply:
Make sure it's a good one. The cheapies are no fun to use.